package.json

```diff
@@ -15,8 +15,13 @@
     "build:js": "webpack --mode production",
     "build:backends": "dash-generate-components ./src/lib/components dash_textarea_autocomplete -p package-info.json --r-prefix '' --jl-prefix ''",
     "build:backends-activated": "(. venv/bin/activate || venv\\scripts\\activate && npm run build:py_and_r)",
-    "build": "npm run build:js && npm run build:backends",
-    "build:activated": "npm run build:js && npm run build:backends-activated"
+    "build:create-tar-file-from-deps": "tar -czf ./deps.tar.gz ./deps && mv ./deps.tar.gz ./deps",
+    "build": "npm run build:js && npm run build:backends && npm run build:create-tar-file-from-deps",
+    "build:activated": "npm run build:js && npm run build:backends-activated",
+    "postbuild:fixup-external-url-jl": "sed -i 's/https:\\/\\/unpkg.com\\/dash_textarea_autocomplete/https:\\/\\/unpkg.com\\/dash-textarea-autocomplete/' src/DashTextareaAutocomplete.jl",
+    "postbuild:fixup-dash-version-project-jl": "sed -i 's/Dash = \"0.1.3\"/Dash = \"0.1.3, 1.0\"/' Project.toml",
+    "postbuild:fixup-resources-path-jl": "sed -i 's/realpath(joinpath( @__DIR__, \"..\", \"deps\"))/artifact\"dash_textarea_autocomplete_resources\"/' src/DashTextareaAutocomplete.jl",
+    "postbuild": "npm run postbuild:fixup-external-url-jl && npm run postbuild:fixup-dash-version-project-jl && npm run postbuild:fixup-resources-path-jl"
```

Comment:
Very nice. Keeping the "post build" fixups separated from the `npm run bulld` make things very clear.

We should add a line mentioning `npm run postbuild` in the release instructions:

https://github.com/etpinard/dash-textarea-autocomplete/blob/main/CONTRIBUTING.md?plain=1#L61

Reply:
We should probably also mention that because of the use of `sed` in the build and that wierd `''` name prefix bug I observed on Windows, this project can only be built on a Linux distribution

Reply:
> this project can only be built on a Linux distribution

Yep, good call. It probably works on Macs too, but I haven't tried :stuck_out_tongue:

Reply:
Isn't MacOs just a linux distro anyways ? 😛